tika/server_test.go

```diff
@@ -244,29 +244,12 @@ func TestAddJavaProps(t *testing.T) {
 		t.Fatalf("NewServer got error: %v", err)
 	}
 
-	err = s.AddJavaProp("foo", "bar")
-	if err != nil {
-		t.Error(err)
-	}
+	s.Javaprops["java.io.tmpdir"] = "/tmp/stuff"
 
 	want := 1
-	got := len(s.javaprops)
+	got := len(s.Javaprops)
```

Comment:
Is there a way we can test that the arguments are correctly added to the `java` command? Would hopefully catch the overwriting `props` every time bug.